frontend/javascripts/admin/tasktype/recommended_configuration_view.js

```diff
@@ -31,6 +31,7 @@ const recommendedConfigByCategory = {
   dataset: {
     fourBit: false,
     interpolation: true,
+    segmentationOpacity: 80,
```

Comment:
```suggestion
    segmentationOpacity: 0,
```

The previous default was 0 according to git and https://discuss.webknossos.org/t/settings-via-task-type/1173/4.